packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-items/jurisdiction-modal-card/jurisdiction-card.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
```

Comment:
This component is nested in too many levels by right `jurisdiction-modal/{component_related_to_it}` should be good enough.